concourse/steps/component_descriptor.mako

```diff
@@ -170,11 +170,21 @@ base_descriptor_file_v2 = os.path.join(descriptor_out_dir, 'base_component_descr
 with open(base_descriptor_file_v2, 'w') as f:
   f.write(dump_component_descriptor_v2(base_descriptor_v2))
 
+# dump base_archive_v2 as valid component archive and pass it to descriptor script
+base_archive_dir = os.path.join(descriptor_out_dir, 'base_component_archive')
+os.makedirs(base_archive_dir, exist_ok=True)
+base_archive_descriptor_file = os.path.join(
+  descriptor_out_dir,
+  component_descriptor_fname(schema_version=gci.componentmodel.SchemaVersion.V2)
+)
+with open(base_archive_descriptor_file, 'w') as f:
+  f.write(dump_component_descriptor_v2(base_descriptor_v2))
```

Comment:
could you not simply cp (or rather symlink/hardlink) the file from `base_descriptor_file_v2`)?

Reply:
added now a `os.link`